src/index.ts

```diff
@@ -1,8 +1,18 @@
 import ejs from "ejs";
-import { getContext, helpers, Import, Export, Property, Schema, XtpSchema } from "@dylibso/xtp-bindgen";
+import { getContext, helpers, Import, Export, Property, Schema, XtpSchema, XtpNormalizedType, ArrayType, BufferType, ObjectType, EnumType, MapType } from "@dylibso/xtp-bindgen";
 
 function toPascalCase(s: string) {
```

Comment:
should we move this to xtp-bindgen? i noticed we do have a `snakeToPascal` helper

Reply:
nice! I am going to use that